README.rst

```diff
@@ -55,6 +57,8 @@ Endpoint                                 Description
 `/forms/post`_                           HTML form that submits to */post*
 `/xml`_                                  Returns some XML
 `/encoding/utf8`_                        Returns page containing UTF-8 data.
+`/file/:path?name=filename`              Creates a new file with *name* in *path* directory.  Creates intermediate directories if necessary.  If *name* is not provided, a unique name is assigned.
+`/file/:path`                            Returns the content of all files in *path* directory as a multipart/form-data response
```

Comment:
seems like it would be more consistent if you had a URL scheme like:

- `/file/:path/:filename`
- `/file/:path/`